config/initializers/semantic_logger.rb

```diff
@@ -14,5 +14,5 @@
   end
 
   log_stash = LogStashLogger.new(Settings.logstash.to_h.merge(customize_event: logstash_formatter))
-  SemanticLogger.add_appender(logger: log_stash, level: :info, formatter: :json)
+  SemanticLogger.add_appender(logger: log_stash, level: :info, formatter: :one_line)
```

Comment:
I'm not sure this is the change we want, have you tested this? You can test by setting the appropriate env vars to log to Logit. But ... I expect that we need `:json` here, to interface with logstash.

What I'm trying to say, and I'm sorry I haven't been clear, is when a request comes into Rails, `semantic-logger` takes all the log events that normally get emitted, and turn them into a single event with all the information. To borrow from the [Semantic Logger Rails page](https://rocketjob.github.io/semantic_logger/rails.html) I mentioned above, this means that this:

![Normal Rails Logs](https://rocketjob.github.io/semantic_logger/images/rails_default.png)

Gets turned into this:

![Semantic Logger Rails Logs](https://rocketjob.github.io/semantic_logger/images/rails_single_line.png)

And likewise, instead of sending a bunch of events to Logit, it just sends one. What I wanted us to spend a little bit of times on is to see if we can replicate this for Sidekiq so that we send one log event with just the info we need to Logit for each job that gets processed. It's a little pie-in-the-sky, but I figured that with the prior-art in Semantic Logger we just might be able to figure this out.